src/librbd/Operations.cc

```diff
@@ -358,6 +359,60 @@ void Operations<I>::execute_rebuild_object_map(ProgressContext &prog_ctx,
 }
 
 template <typename I>
+int Operations<I>::check_object_map(ProgressContext &prog_ctx) {
+  CephContext *cct = m_image_ctx.cct;
+  ldout(cct, 5) << this << " " << __func__ << dendl;
+  int r = m_image_ctx.state->refresh_if_required();
+  if (r < 0) {
+    return r;
+  }
+
+  r = invoke_async_request("check object map", true,
+                           boost::bind(&Operations<I>::check_object_map, this,
+                                       boost::ref(prog_ctx), _1),
+			   [] (Context *c) { c->complete(-EOPNOTSUPP); });
```

Comment:
It would be good to add the necessary RPC plumbing to support this when the exclusive lock cannot be acquired -- otherwise you'd never be able to verify the object map on opened images.